test/image.bats

```diff
@@ -110,9 +110,6 @@ function teardown() {
 	imageid=$(crictl images --quiet "$IMAGE:go")
 	[ "$imageid" != "" ]
 
-	output=$(crictl images --quiet @"$imageid")
```

Comment:
Note all these changes: the `@`_id_ syntax is no longer recognized for a “filter”, with the minimal version of this PR.

Alternatively, the new code could explicitly recognize and support the `@`_id_ syntax.

(Or, going the other extreme, all of these test commands could use `crictl inspecti` instead.)